babelConfig.js

```diff
@@ -19,6 +21,11 @@ module.exports = function (options = {}) {
           'corejs': '3.13.0',
           // a lot of tests use sinon.stub & others that stopped working on ES6 modules with webpack 5
           'modules': options.test ? 'commonjs' : 'auto',
+          targets: isES6 ? {
```

Comment:
We're managing the target here

https://github.com/prebid/Prebid.js/blob/1a8b99330e53b4ae50f367d6a764a2f7dddb4f68/package.json#L29

Changing it would be breaking

Reply:
Yeah, we noticed as well. @Shadialsayed is one my engineers and we are going to use this in our infrastructure soon.

The `package.json` config is too static. And because this change only overrides the `package.json` if there's the `es6` build cli parameter, it isn't a breaking change. My argument would be that the `package.json` is the default, which can be overriden by build parameters.

Reply:
this appears to fallback to the 1 percent target

Reply:
this has now changed the target to match, but now keeping them in sync seems difficult. Can we import a constant?

Reply:
We are doing some more tests, but it seems that the output bundle is still a lot smaller, even the target is the same.

If it works without specifying a target here, we'll remove it here.

Reply:
I'm a bit confused, looking forward to your tests